parboiled-core/src/main/scala-3/org/parboiled2/ParserMacros.scala

```diff
@@ -334,6 +335,72 @@ object ParserMacros {
   import scala.quoted._
   import scala.compiletime._
 
+  // TODO: the `R` type parameter is a workaround for https://github.com/lampepfl/dotty/issues/13376
+  // Discussion at https://github.com/sirthias/parboiled2/pull/274#issuecomment-904926294
+  def runImpl[L <: HList: Type, R: Type]()(ruleExpr: Expr[RuleN[L]], schemeExpr: Expr[Parser.DeliveryScheme[L]])(using
+      Quotes
+  ): Expr[R] =
+    import quotes.reflect.*
```

Comment:
We haven't used indentation-based syntax so far. I would prefer to keep braces for now (my IntelliJ gets completely messed up without), though it's not a strong opinion.

Reply:
OK, sure. I'll add the braces.